internal/pkg/deploy/cloudformation/stack/scheduled_job.go

```diff
@@ -196,6 +196,7 @@ func (j *ScheduledJob) Template() (string, error) {
 		Publish:                  publishers,
 		Platform:                 convertPlatform(j.manifest.Platform),
 		EnvVersion:               j.rc.EnvVersion,
+		Version:                  j.rc.Version,
```

Comment:
do we need to add the flags to `job deploy`? or is that for another PR

Reply:
yeah another PR this one is big enough